src/typingsInstallerCore/nodeTypingsInstallerCore.ts

```diff
@@ -0,0 +1,200 @@
+namespace ts.server.typingsInstaller {
```

Comment:
Why not move this to `typingsInstallerCore\tsconfig.json` project since that's the one that exposes api?

Reply:
nodeTypingsInstallerCore.ts is in typingsInstallerCore now. And both typingsInstaller/typingsInstallerCore use the same namespace, typingsInstaller. 

I guess you are referring to something else, though. Can you tell me what it is?

Reply:
Ok I don't know what I saw but this is already what you have done. :)